clusterctl/examples/vsphere/generate-yaml.sh

```diff
@@ -53,11 +53,11 @@ fi
 
 OS=$(uname)
 if [[ "$OS" =~ "Linux" ]]; then
-  MACHINE_CONTROLLER_SSH_PUBLIC=$(cat $MACHINE_CONTROLLER_SSH_HOME$MACHINE_CONTROLLER_SSH_PUBLIC_FILE|base64 -w0)
-  MACHINE_CONTROLLER_SSH_PRIVATE=$(cat $MACHINE_CONTROLLER_SSH_HOME$MACHINE_CONTROLLER_SSH_PRIVATE_FILE|base64 -w0)
+  MACHINE_CONTROLLER_SSH_PUBLIC=$(cat $MACHINE_CONTROLLER_SSH_HOME$MACHINE_CONTROLLER_SSH_PUBLIC_FILE | base64 | tr -d '\r\n')
```

Comment:
The original reason to introduce the OS check here is to handle the lack of -w0 option on Mac.
Seems like by using "tr -d '\r\n'", we can get rid of the OS check completely as the Linux codepath is the same as the Mac codepath.

Reply:
Good point. I just did a search/replace, but collapsing to a single set of commands that work for both is definitely better.